src/main/java/unrc/dose/UserProfileEndpoint.java

```diff
@@ -0,0 +1,108 @@
+                ),
+            (q, a) -> LOGEER.info("Logging Received request for UserProfile Rest API"))
+        
+           .post(
+                   path("/:id/profile")
+                       .withDescription("creates a new user profile")
+                        .withQueryParam()
+                            .withName("displayName")
+                            .withDescription("user profile display name").and()
+                       .withQueryParam()                            
+                       .withName("twitterId")
+                       .withDescription("user profile twitter id").and()
+                      
+                        .withResponseType(String.class),
+                    (req, res) -> {
+                    	if (!User.exists(Integer.parseInt(req.params("id")))) {
+           					throw new UserNotFoundException(String.valueOf(Integer.parseInt(req.params("id"))));
+          					}
+                    	
+                        return UserProfileService.userprofile(
+                        Integer.parseInt(req.params("id")),req.queryParams("displayName"),req.queryParams("twitterId"));
+                    	
+                 }
+                )
+             .put(
+                   path("/:id/profile")
+                       .withDescription("Modify user profile")
+                       .withQueryParam()
+                       .withName("displayName")
+                       .withDescription("user profile display name").and()
+                       .withQueryParam()
+                    	.withName("twitterId")
+                       	.withDescription("user profile twitter id").and()  
+                        .withResponseType(String.class),
+                     
+                    (req, res) -> {
+          					if (!User.exists(Integer.parseInt(req.params("id")))) {
+           					throw new UserNotFoundException(String.valueOf(Integer.parseInt(req.params("id"))));
+          					}
+          					UserProfile up = (UserProfile) UserProfile.find("user_id = ?", req.params("id")).get(0);
+          				    up.setDisplayName (req.queryParams("displayName"));
+                            up.setTwitterId (req.queryParams("twitterId"));
+                            up.saveIt();
+                            return up.toJson(true, "user_id", "displayName","twitterId");
+                            }
+                )
+                .get(
+                       path("/:id/profile")
+                            .withDescription("Will return the user profile of user's id")
+                             .withPathParam()
+                                .withName("displayName")
+                                .withDescription("user profile display name").and()
+                            .withPathParam()
+                            	.withName("twitterId")
+                            	.withDescription("user profile twitter id").and()
+                            .withResponseType(String.class),
```

Comment:
The get should return a `UserProfile` (as JSON), and as such it should not have any parameters